tlatools/org.lamport.tlatools/src/tlc2/tool/Simulator.java

```diff
@@ -302,6 +302,8 @@ public int simulate() throws Exception {
 			return MP.printError(EC.TLC_NO_STATES_SATISFYING_INIT_AND_CONSTRAINT);
 		}
 
+		MP.printMessage(EC.TLC_INIT_GENERATED1, String.valueOf(this.numOfGenStates.longValue()), "");
+
```

Comment:
Should we get the number of distinct states from the data structures `initStates` rather than from a statistics variable?